packages/react-instantsearch/src/core/createInstantSearchManager.js

```diff
@@ -29,6 +29,9 @@ export default function createInstantSearchManager({
   helper.on('result', handleSearchSuccess);
   helper.on('error', handleSearchError);
 
+  let derivedHelpers = {};
```

Comment:
If I remove this default value, then, I need to add this " if (derivedHelpers) " when looking for detaching helpers for the first time.